crates/apub/src/api/test.rs

```diff
@@ -0,0 +1,42 @@
+use activitypub_federation::config::Data;
+use lemmy_api_common::context::LemmyContext;
+use lemmy_db_schema::{
+  source::{
+    instance::Instance,
+    local_user::{LocalUser, LocalUserInsertForm},
+    person::{Person, PersonInsertForm},
+  },
+  traits::Crud,
+};
+use lemmy_db_views::structs::LocalUserView;
+use lemmy_utils::error::LemmyResult;
+
+#[derive(Default)]
+pub struct TestUser {
+  pub name: String,
+  pub bio: Option<String>,
+  pub admin: bool,
+}
```

Comment:
You can probably remove `name` and generate a random string instead. For bio use `Option<&'static str>` then you dont need to call .to_string().

Reply:
is there an example somewhere for generating random names that are safe to not collide?

Reply:
uuid should be fine: `let x = uuid::Uuid::new_v4().to_string()`